source/software/r_package_management.rst

```diff
@@ -6,51 +6,106 @@ R package management
 Introduction
 ------------
 
-Most of the useful R packages can be installed separately. Some of those are 
-part of the centrally installed R modules. However, given the astounding number of 
-packages, it is not sustainable to install each and everyone of them system wide. 
-Fortunately, it is very easy for users to install R packages themselves.
-If you do encounter problems when doing so, do not hesitate to contact support.
+There exist thousands of R packages, available from online repositories like CRAN,
+Bioconductor or github. Depending on the R version, the more commonly used packages like `ggplot2`, `tidyverse` or `readr` 
+are either already included in the centrally installed R module or can be accessed by
+loading the R-bundle-CRAN and R-bundle-Bioconductor modules, e.g.:
+
+.. code:: r
+
+       $ module load R-bundle-Bioconductor/3.16-foss-2022b-R-4.2.2    
+
+It is possible, however, that these modules do not contain all R packages you need
+or that the package versions do not meet your requirements. In this case you will
+need to locally install those packages, as will be described below. Do not hesitate
+to contact support when encountering issues during these local installations.
 
 .. _r_package_management_standard_lib:
 
 Standard R package installation
 -------------------------------
 
-Setting up your own package repository for R is straightforward.
+Firstly, it is important to realize that R will use by default the `$VSC_HOME/R` path
+to install new packages. Since home directories have limited quota, it is not
```

Comment:
..., they are not ...

Reply:
Agreed. But solved this missmatch slightly differently:
Since `$VSC_HOME` has limited quote, it is not..."